### DISCUSSION TITLE
Linear path editing

### DISCUSSION THREAD

**Discussion by mjakubowskilkr** [freeplane/freeplane#1070] [Help] [answered]
Hello, I've started to using connectors in mind map and I've got 2 questions:
- can I add new point to linear path, it would be helpful to organize mind map better:
- can I change linear path line - when nodes are close, the path is very long and I don't know how to change it.
Thanks in advance.

**Comment by macmarrum** [comment, marked as answer]
* I believe **Linear path** has a fixed number of elbows, i.e. 2
* Please have a look at https://docs.freeplane.org/attic/new-wiki-initiative/understanding-nodes.html#connectors
